backend-api/src/functions/asyncActiveSession/tokenService/tokenService.ts

```diff
@@ -32,37 +30,6 @@ export class TokenService implements ITokenService {
 
     const jwks = stsJwksEndpointResponseResult.value;
 
-    const getJweComponentsResult = this.getJweComponents(jwe);
```

Comment:
Logic moved to its own class

Reply:
We can delete getJweComponents and decryptJwe functions from this file now, they are unused

Reply:
I thought I had done that! What's going on... 👀

Reply:
I think this came from the rebase 😢

Reply:
Thanks for picking up on that James. I have deleted them again.